app/src/main/res/layout/item_movie.xml

```diff
@@ -27,20 +27,21 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@id/item_movie_iv"
         app:layout_constraintTop_toTopOf="@+id/item_movie_iv"
-        tools:text="title" />
+        tools:text="Title" />
```

Comment:
don't you think that using string resources would be nicer?

Reply:
@aviramp No need , as we are talking about "tools" namespace

Reply:
Yeah, you are right.